media/js/base/core-datalayer.js

```diff
@@ -27,7 +27,7 @@ if (typeof window.Mozilla.Analytics === 'undefined') {
         if (!isModernBrowser) {
             return 'false';
         }
-        return document.querySelector('[data-download-os]') !== null
+        return document.querySelector('[data-download-version]') !== null
```

Comment:
This file will be removed in #14913 but the test needs to pass in this PR for now.